### PR TITLE
Translation to Spanish

### PR DESCRIPTION
**Pull request by hiperterminal**
I had been working in the Spanish translation of Zettlr-docs. It's not over, but I want to try a pull request before to see how does it work and share what I have done. I finished:

- Welcome
- Concepts
- Core
- Academic tools

_First steps_ needs some translations and all _References_ section, but I hope this is good enough to start.

### CODE REVIEW

File: assets/index.html [hunk 1]
```diff
@@ -141,6 +141,7 @@ <h1 id="main-head">
         <li><a href="/ja" class="button-link"><span class="flag">🇯🇵</span> 日本語</a></li>
         <li><a href="/it" class="button-link"><span class="flag">🇮🇹</span> Italiano</a></li>
         <li><a href="/fr" class="button-link"><span class="flag">🇫🇷</span> Français</a></li>
+        <li><a href="/es" class="button-link"><span class="flag">ES</span> Español</a></li>
```
Comment (nathanlesage) [inline assets/index.html:144]: Here please use the corresponding flag emoji: 🇪🇸

**Review by nathanlesage — COMMENTED** [review]
Almost perfect, just tried it out and it looks and works well! There's just a very small improvement to use the corresponding flag emoji; I've commented that emoji so you can simply copy it from the comment and insert it.

Then we can merge and have another translation :)

File: assets/index.html (hunk 1 above)
Comment (hiperterminal) [reply assets/index.html]: Emoji included. Many thanks!

**Comment by nathanlesage** [comment]
Looks perfect! Thank you! I'll be merging the PR now. In case you want to add more or spot some mistakes, don't worry, just open new PRs, which I can then merge as well! :)

**Pull request merged**